packages/sdk/src/clients/multicall/client.e2e.ts

```diff
@@ -5,19 +5,24 @@ import { ethers } from "ethers";
 import { emp } from "..";
 
 // multicall contract deployed to mainnet
-const address = "0xeefba1e63905ef1d7acba5a8513c70307c1ce441";
+const multicallV1Address = "0xeefba1e63905ef1d7acba5a8513c70307c1ce441";
+const multicallV2Address = "0x5ba1e12693dc8f9c48aad8770482f4739beed696";
```

Comment:
again, i think you were right to want to create a separate client, when we talked about it i misunderstood which client you were referring to.  part of the reason we should separate it is because we need to have slightly different tests for v2, and it i think it makes more sense to keep that distinct ffrom the v1 tests.